draft-ietf-quic-transport.md

```diff
@@ -2999,10 +2999,10 @@ appended at the end.
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 |                       ACK Block Count (i)                   ...
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
```

Comment:
Can you move the ACK Block Count down as well.  It's odd to have that up here.

Reply:
Also, how about putting the ECN info in this diagram, rather than having it in a separate diagram?

Reply:
Yes, I merged the ECN counters and text into a single section for the ACK_ECN frame. I think that is very reasonable. So please check this latest change.